### ISSUE TITLE
Docker environment setup

### ISSUE THREAD

**Issue by gabiudrescu** [Monofony/SymfonyStarter#193]
**Describe the proposed solution**
We should have a docker setup configured for a oneline setup of this project; eg: docker-compose up should bring the application with full configuration online.

**Comment by loic425** [comment]
For developments, the new Symfony binary is better than a docker setup now. But we can create a docker setup for production environment.

**Comment by gabiudrescu** [comment]
afaik, Symfony binary is providing only PHP, not the entire stack required for a modern app: MySQL, Redis, queuing, nodejs for frontend etc.

and I remember Jedisjeux had a docker setup (https://github.com/Jedisjeux/Docker/tree/master). I guess it was included in the initial Monofony repo in the days, right?

anyway, I've put this issue here to make sure we don't forget about it - in an attempt to create a backlog of features and a place for debate.

**Comment by loic425** [comment]
@gabiudrescu You're right, it's cool to init a backlog. I just want to prevent adding to much features on this starter. Cause it takes so much time to remove all you don't need on a starter.
We don't need Redis or Elasticsearch or whatever else on every projects...
At Mobizel where we use Monofony to build almost every Symfony projects, @smildev uses a mysql docker instance for every projects with the same mysql version. Personally, I have one local mysql and prefer use it rather than a docker instance to keep it simpler. 
What I mean is nowadays, Docker is not useful on every Symfony projects for development.

@lchrusciel @smildev What's your opinion?

**Comment by lchrusciel** [comment]
I would say that it is hard to decide what should be inside. But some links/docs about available docker images could be useful. 

Maybe we should support simplified docker setup with PHP and mysql only?
